scripts/addr2line.py

```diff
@@ -263,27 +287,27 @@ def get_prefix(s):
             #print(f">>> '{line}': None")
             return None
 
-    def __init__(self, executable, kallsyms='/proc/kallsyms', before_lines=1, context_re='', verbose=False, concise=False, cmd_path='addr2line'):
+    def __init__(self, executable: str, kallsyms: str = '/proc/kallsyms', before_lines: int = 1, context_re: Optional[str] = '', verbose: bool = False, concise: bool = False, cmd_path: str = 'addr2line'):
         self._executable = executable
         self._kallsyms = kallsyms
-        self._current_backtrace = []
+        self._current_backtrace : list[tuple[str, str]] = []
```

Comment:
nit, might want to put `:` next to `self._current_backtrace` , like:
```py
self._current_backtrace: list[tuple[str, str]] = []
```

Reply:
good eye, fixed! (even more reason for black :))